packages/meta-extraction/test/test.js

```diff
@@ -32,15 +32,15 @@ describe.only('Abstract Extraction', function () {
     assert.equal(pdf,pdfExpected);
   });
 
-  it.skip('should extract abstract and pdf using aaai rule (aaai.org)', async function () {
+  it('should extract abstract and pdf using aaai rule (aaai.org)', async function () {
```

Comment:
Why are we skipping AAAI tests?

Reply:
i think the change to from skip to not skip

Reply:
Oh, I completely missed that. Thanks!